src/form/lpac.jl

```diff
@@ -130,3 +130,254 @@ function constraint_ohms_yt_to(pm::AbstractLPACCModel, n::Int, f_bus, t_bus, f_i
+    q_dc = get(var(pm, n), :q_dc, Dict()); _check_var_keys(q_dc, bus_arcs_dc, "reactive power", "dcline")
+    p_ne = get(var(pm, n), :p_ne, Dict()); _check_var_keys(p_ne, bus_arcs_ne, "active power", "ne_branch")
+    q_ne = get(var(pm, n), :q_ne, Dict()); _check_var_keys(q_ne, bus_arcs_ne, "reactive power", "ne_branch")
+
+    cstr_p = JuMP.@constraint(pm.model,
+        sum(p[a] for a in bus_arcs)
+        + sum(p_dc[a_dc] for a_dc in bus_arcs_dc)
+        + sum(psw[a_sw] for a_sw in bus_arcs_sw)
+        + sum(p_ne[a] for a in bus_arcs_ne)
+        ==
+        sum(pg[g] for g in bus_gens)
+        - sum(ps[s] for s in bus_storage)
+        - sum(pd for pd in values(bus_pd))
+        - sum(gs for gs in values(bus_gs))*(1 + 2*phi)
+    )
+    cstr_q = JuMP.@constraint(pm.model,
+        sum(q[a] for a in bus_arcs)
+        + sum(q_dc[a_dc] for a_dc in bus_arcs_dc)
+        + sum(qsw[a_sw] for a_sw in bus_arcs_sw)
+        + sum(q_ne[a] for a in bus_arcs_ne)
+        ==
+        sum(qg[g] for g in bus_gens)
+        - sum(qs[s] for s in bus_storage)
+        - sum(qd for qd in values(bus_qd))
+        + sum(bs for bs in values(bus_bs))*(1 + 2*phi)
+    )
+    if _IM.report_duals(pm)
+        sol(pm, n, :bus, i)[:lam_kcl_r] = cstr_p
+        sol(pm, n, :bus, i)[:lam_kcl_i] = cstr_q
+    end
+end
+
+""
+function constraint_ne_ohms_yt_from(pm::AbstractLPACCModel, n::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_fr, b_fr, tr, ti, tm, vad_min, vad_max)
+    p_fr = var(pm, n,    :p_ne, f_idx)
+    q_fr = var(pm, n,    :q_ne, f_idx)
+    phi_fr = var(pm, n, :phi_fr_ne, i)
+    phi_to = var(pm, n, :phi_to_ne, i)
+    va_fr  = var(pm, n, :va_fr_ne, i)
+    va_to  = var(pm, n, :va_to_ne, i)
+    cs     = var(pm, n, :cs_ne, i)
+    z = var(pm, n, :branch_ne, i)
+    JuMP.@constraint(pm.model, p_fr ==  (g+g_fr)/tm^2*(z + 2*phi_fr) + (-g*tr+b*ti)/tm^2*(cs + phi_fr + phi_to) + (-b*tr-g*ti)/tm^2*(va_fr-va_to) )
+    JuMP.@constraint(pm.model, q_fr == -(b+b_fr)/tm^2*(z + 2*phi_fr) - (-b*tr-g*ti)/tm^2*(cs + phi_fr + phi_to) + (-g*tr+b*ti)/tm^2*(va_fr-va_to) )
+end
+
+""
+function constraint_ne_ohms_yt_to(pm::AbstractLPACCModel, n::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm, vad_min, vad_max)
+    p_to = var(pm, n,    :p_ne, t_idx)
+    q_to = var(pm, n,    :q_ne, t_idx)
+    phi_fr = var(pm, n, :phi_fr_ne, i)
+    phi_to = var(pm, n, :phi_to_ne, i)
+    va_fr  = var(pm, n, :va_fr_ne, i)
+    va_to  = var(pm, n, :va_to_ne, i)
+    cs     = var(pm, n, :cs_ne, i)
+    z = var(pm, n, :branch_ne, i)
+    JuMP.@constraint(pm.model, p_to ==  (g+g_to)*(z + 2*phi_to) + (-g*tr-b*ti)/tm^2*(cs + phi_fr + phi_to) + (-b*tr+g*ti)/tm^2*-(va_fr-va_to) )
+    JuMP.@constraint(pm.model, q_to == -(b+b_to)*(z + 2*phi_to) - (-b*tr+g*ti)/tm^2*(cs + phi_fr + phi_to) + (-g*tr-b*ti)/tm^2*-(va_fr-va_to) )
+end
+
+
+function ref_calc_angle_difference_bounds(buspairs, conductor::Int=1)
```

Comment:
This function should be placed in `src/base/ref.jl`.  However, I don't think you need this function, see previous note.

Reply:
Function not used?  Can be removed.